testsuite/integration-arquillian/tests/base/src/test/resources/authorization-test/enforcer-config-claims-provider.json

```diff
@@ -0,0 +1,81 @@
+{
+  "realm": "test-realm-authz",
+  "realm-public-key": "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCrVrCuTtArbgaZzL1hvh0xtL5mc7o0NqPVnYXkLvgcwiC3BjLGw1tGEGoJaXDuSaRllobm53JBhjx33UNv+5z/UMG4kytBWxheNVKnL6GgqlNabMaFfPLPCF8kAgKnsi79NMo+n6KnSY8YeUmec/p2vjO2NjsSAVcWEQMVhJ31LwIDAQAB",
+  "auth-server-url": "http://localhost:8180/auth",
+  "ssl-required": "external",
+  "resource": "test-app-authz",
+  "bearer-only": true,
+  "credentials": {
+    "secret": "secret"
+  },
+  "policy-enforcer": {
+    "paths": [
+      {
+        "path": "/claims-provider",
+        "methods": [
+          {
+            "method": "POST",
+            "scopes": [
+              "create"
+            ]
+          }
+        ],
+        "claim-information-point": {
+          "claims": {
+            "claim-from-request-parameter": "{request.parameter['a']}",
+            "claim-from-header": "{request.header['b']}",
+            "claim-from-cookie": "{request.cookie['c']}",
+            "claim-from-remoteAddr": "{request.remoteAddr}",
+            "claim-from-method": "{request.method}",
+            "claim-from-uri": "{request.uri}",
+            "claim-from-relativePath": "{request.relativePath}",
+            "claim-from-secure": "{request.secure}",
+            "claim-from-json-body-object": "{request.body['/a/b/c']}",
+            "claim-from-json-body-array": "{request.body['/d/1']}",
+            "claim-from-body": "{request.body}",
+            "claim-from-static-value": "static value",
+            "claim-from-multiple-static-value": ["static", "value"]
+          }
+        }
+      },
+      {
+        "path": "/http-post-claim-provider",
+        "claim-information-point": {
+          "http": {
+            "url": "http://localhost:8989/post-claim-information-provider",
+            "method": "POST",
```

Comment:
if POST is supported and not only GET, there should be an option to set also the bofy

Reply:
You mean, support a property which you can set the body when using POST ? FYI, if using POST you can already use the configuration to define request parameters.

Reply:
yes, add a property in the configuration to specify the POST request body (with placeholders). 
At the moment, as I understand, the only option to set the POST request body is by "parameters", which are sent as the body in the format of url-encoded params (i.e., A=1&b=2&c=3 )

Reply:
I'm not sure about this one. Do you really think parameters/headers are not enough to query the external service ?